src/screen.rs

```diff
@@ -1103,48 +1074,64 @@ impl Screen {
             need_clear_screen = true;
         }
 
-        // Output the left prompt if it has changed.
-        if self.scrolled() && !is_final_rendering {
-            self.r#move(0, 0);
-            self.write_command(ClearToEndOfLine);
```

Comment:
Re f154bc2c7bb (Instead of prompt line_breaks track line_starts, 2025-10-14)
which does

```diff
-                for (i, &line_break) in left_prompt_layout.line_breaks.iter().enumerate() {
+                for (i, &next_line) in left_prompt_layout.line_starts.iter().enumerate() {
                     self.write_command(ClearToEndOfLine);
```

shouldn't it be `iter().skip(1).enumerate()` to avoid doing

	ClearToEndOfLine
	Osc133PromptStart
	ClearToEndOfLine

which would actually (and surprisingly) break the prompt marker on some terminals, see
https://github.com/tmux/tmux/issues/4183